torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -60,49 +46,168 @@ bool validateDomain(TensorView* tv, TensorDomain* new_td) {
       first_mismatch >= (int)tv->getComputeAtPosition();
 }
 
+// Return the max position in consumer that producer can be inlined to
+// Cannot inline:
+//   Reduction dimensions in producer
+//   Block broadcast dimensions in producer
+//   Vectorized dimensions in producer or consumer
+//   Dimensions derived from root dimensions that exist in both but are
+//   unmappable
 unsigned int getReplayablePosPasC(
```

Comment:
nit: Could be nice if the `PasC` and `CasP` functions could be combined into a single function for saving repeated code, but simply having two separate functions might be easier to understand.

Reply:
I agree, though I generally really like the syntax of `PasC` and `CasP` as it's a common pattern and often implementation is slightly different. Let me know though if you really think we should/can merge them cleanly.